pkg/nfd-master/nfd-master.go

```diff
@@ -572,15 +573,6 @@ func addAnnotations(n *api.Node, annotations map[string]string) {
 	}
 }
 
-func updateMap(input map[string]int32) map[string]int {
```

Comment:
We shouldn't be removing this function as it is used [here](https://github.com/swatisehgal/node-feature-discovery/blob/rte/pkg/nfd-master/nfd-master.go#L599). Please refer to [this](https://github.com/swatisehgal/node-feature-discovery/pull/1#pullrequestreview-534225565) comment on why we needed this in the first place.

Reply:
reverted